TestSwitchApi/TestSwitchApi/TestSwitchApi.csproj

```diff
@@ -7,6 +7,7 @@
 
     <ItemGroup>
       <PackageReference Include="EntityFramework" Version="6.4.0" />
+      <PackageReference Include="FluentAssertions" Version="5.10.3" />
```

Comment:
I don't think we need this in the main project?
Should it just be in the test project?